pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -419,4 +419,14 @@
   footer_menu_contact: 'Contact us'
   footer_menu_support: 'Support'
   footer_menu_terms: 'Terms'
-  footer_menu_privacy: 'Privacy Policy'
\ No newline at end of file
+  footer_menu_privacy: 'Privacy Policy'
+
+  learn_banner_heading: 'HOUR of CODE'
```

Comment:
Ideally we want to keep it all uppercase but style the 'of' to be smaller. But then we might have to break up the string? Either way, not blocking merge (todo for later).